vm/src/stdlib/time_module.rs

```diff
@@ -58,6 +58,18 @@ pub fn get_time() -> f64 {
     Date::now() / 1000.0
 }
 
+#[cfg(any(not(target_arch = "wasm32"), target_os = "wasi"))]
+pub fn get_time_ns() -> u64 {
```

Comment:
In general, we never want the interpreter is crashed by python code. The correct signature of failible function is like this:
```suggestion
pub fn get_time_ns() -> PyResult<u64> {
```